dali/operators/reader/numpy_reader_op.cc

```diff
@@ -63,6 +174,12 @@ the list of files in the sub-directories of `file_root`.)code", "*.npy")
       R"code(If true, reader shuffles whole dataset after each epoch. It is exclusive with
 `stick_to_shard` and `random_shuffle`.)code",
       false)
+  .AddOptionalArg<int>("anchor", R"code(Specifies the anchor for sliced reads.\n
+If no anchor is specified, but shape is set, anchor=0 is assumed.)code",
+      std::vector<int>(), true)
+  .AddOptionalArg<int>("shape", R"code(Specifies the shape of the slice for sliced reads.\n
+If no shape is specified, the whole file is read.)code",
+      std::vector<int>(), true)
```

Comment:
BTW - maybe it is a good opportunity to add `file_list` support to this operator?

Reply:
file list means: the user specifies a list with files and it just uses those? like:

dir1/file1.npy
dir2/file2.npy
dir2/file3.npy
...

so if I find the file list, I will just use that?

Reply:
Yes, that is what I mean.

Reply:
added one.